.claude/skills/skill-creator/scripts/run_eval.py

```
@@ -0,0 +1,310 @@
+    parser.add_argument("--model", default=None, help="Model to use for claude -p (default: user's configured model)")
+    parser.add_argument("--verbose", action="store_true", help="Print progress to stderr")
+    args = parser.parse_args()
+
+    eval_set = json.loads(Path(args.eval_set).read_text())
+    skill_path = Path(args.skill_path)
+
+    if not (skill_path / "SKILL.md").exists():
+        print(f"Error: No SKILL.md found at {skill_path}", file=sys.stderr)
+        sys.exit(1)
+
+    name, original_description, content = parse_skill_md(skill_path)
```

Ruff (0.15.12)

[warning] Unpacked variable `content` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)